packages/backend-core/src/features/index.ts

```diff
@@ -267,4 +267,5 @@ export class FlagSet<V extends Flag<any>, T extends { [key: string]: V }> {
 // default values set correctly and their types flow through the system.
 export const flags = new FlagSet({
   DEFAULT_VALUES: Flag.boolean(false),
+  [FeatureFlag.ENRICHED_RELATIONSHIPS]: Flag.boolean(false),
```

Comment:
It'll be nice when the frontend is all TypeScript as well because we won't need enums. We'd get full type checking on the literal strings.

Reply:
Yes, it will be. But I think I still prefer enums for these use cases, as it is easier to find references and refactor. Otherwise we have just typed magic strings.

Reply:
What specifically is made easier by the enums? With the typed strings, your editor will suggest to you the correct options after typing `isEnabled(`, and if you delete a flag, TypeScript will error all of the places that flag is checked.